frontend/src/bundles/auth/components/auth-layout/styles.module.scss

```diff
@@ -28,13 +25,13 @@
         display: inline-block;
         width: 75px;
         margin-right: auto;
-        background-color: var(--sunglow);
+        background-color: #ffd231;
```

Comment:
Why did you rremove var(--sunglow)?

Reply:
CI_PR Linter problems, bt-116 introduces var(--sunglow)